docs/source/addons/theme.md

```diff
@@ -119,44 +119,54 @@ or add a key in your `package.json` project:
+5. You can safely delete now your project's `theme` folder, since the one in the add-on will take precedence.
 
-# Modify the theme from another add-on
+## Using your own scape hatch
 
-```{warning}
-This feature works only with scss-based themes, and not with SemanticUI themes.
-```
-
-Volto theming uses SemanticUI theming capabilities to define and extend at a later point if needed a theme. However, while maintaining and playing well with the Semantic UI Volto base, using a traditional CSS approach can be done using the less-based `extras` scape hatch.
+Volto theming uses SemanticUI theming capabilities to define and extend a theme for your site. However, while maintaining and playing well with the Semantic UI Volto base, using a traditional CSS approach can be done using the less-based `extras` scape hatch.
 
 At the same time, one can entirely bail off the extras scape hatch and add your own, by customizing the `theme.js` module in Volto.
 
 ```js
-// These are the default
 import 'semantic-ui-less/semantic.less';
 import '@plone/volto/../theme/themes/pastanaga/extras/extras.less';
 
 // You can add more entry points for theming
 import '@kitconcept/volto-light-theme/theme/main.scss';
 ```
 
-Then you can build your own approach for theming, even use the preprocessor of your choice (here, scss) while maintaining the "base" Volto theme, but customizing it using the resultant CSS from your own theming entry point.
+While building your own scape hatch for theming, you can use the preprocessor of your choice (in the example, scss) while maintaining the "base" Volto theme, but customizing it using the resultant CSS.
 
 You can see an example of such a theme in: https://github.com/kitconcept/volto-light-theme
 
-This approach supports only scss preprocessor, but the same approach can be used to add more entry points or use another preprocessor or theming approach.
+## Modify a custom theme using from another add-on
+
+Sometimes you have a custom theme that you want to reuse through all your projects, but with some differences, maintaining the base.
+Usually, the only option would be to use an addon that adds more CSS to the base theme, using imports that will load after the theme.
+However, there is a problem with this approach.
+You cannot use existing theme variables (like breakpoints, etc) on these new CSS.
+The same way, it gets somewhat detached from the normal flow of the loaded theme.
+Same applies for add-ons, they are detached from the current theme.
+One could use a SemanticUI approach for making this work, but it's SemanticUI bound.
+
+```{warning}
+This is only possible when using your own scape hatch, and works only with scss-based themes, and not with SemanticUI themes, since it enables a couple of entry points that only support scss files. For an example of how it could be used, see: https://github.com/kitconcept/volto-light-theme
```

Comment:
```suggestion
This is only possible when using your own escape hatch, and works only with SCSS-based themes, and not with SemanticUI themes, since it enables a couple of entry points that only support SCSS files.
For an example of how it could be used, see: https://github.com/kitconcept/volto-light-theme
```